help/destinations/catalog/advertising/acxiom-prospect-suppression.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Acxiom Prospect-Suppression
```

Comment:
what are the plans with this destination document? are you planning to have documentation for both Acxiom destinations in the same PR? 

I know that we were discussing the Prospect Suppression destination documentation in the separate PR https://github.com/AdobeDocs/experience-platform.en/pull/312

Reply:
This is a second destination card.  This documentation should be kept independent of the prospect.  While we would like to have both completed completing prospect would come first, as I think they want that card ready for the summit.